internal/indexer/definitions/sharewood.yaml

```
@@ -0,0 +1,69 @@
+---
+name: Sharewood
+identifier: sharewood
+description: Sharewood (Sw) is a Private French Torrent Tracker for MOVIES / TV / GENERAL
+language: fr-fr
+urls:
+  - https://www.sharewood.tv/
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: passkey
+    type: secret
+    required: true
+    label: Passkey
+    help: "Aller sur Mon Profil, puis cliquer sur Afficher Passkey"
```

This should be in English.

```suggestion
    required: true
    label: Passkey
    help: "Go to My Profile, then click on Show Passkey"
```